mappings/net/minecraft/sortme/LevelIndexedProcessor.mapping

```diff
@@ -0,0 +1,73 @@
+CLASS cep net/minecraft/sortme/LevelIndexedProcessor
+	FIELD a maxLevelCount I
+	FIELD b toProcess [Lit/unimi/dsi/fastutil/longs/LongLinkedOpenHashSet;
+	FIELD c levelMap Lit/unimi/dsi/fastutil/longs/Long2ByteFunction;
+	FIELD d minUnprocessedLevel I
+	FIELD e hasUpdates Z
+	METHOD <init> (III)V
+		ARG 1 levelCount
+		ARG 2 initLevelCapacity
+		ARG 3 initTotalCapacity
+	METHOD a updateMinUnprocessed (I)V
+		ARG 1 maxLevel
+		ARG 2 previousMinUnprocessed
+	METHOD a minLevel (II)I
+		ARG 1 level
+		ARG 2 max
+		ARG 3 ret
+	METHOD a process2 (JI)V
+	METHOD a addWithlevel (JII)V
+		ARG 1 id
+		ARG 3 level
+		ARG 4 targetLevel
+	METHOD a removeFromlevel (JIIZ)V
```

Comment:
Extremely minor thing again, the casing here and a few other places is wrong. 😄

Reply:
Ah, I must have missed that. I initially didn't name it level, and then used `sed` to search and replace it, and unintentionally used option to ignore case.